spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
Cool =]. Bear in mind that I checked in the worker task code. So you'll need to merge that change in.